crates/forge/tests/cli/test_helpers.rs

```diff
@@ -0,0 +1,26 @@
+//! Test helpers for Forge CLI tests.
```

Comment:
There is already a `utils` module, please move there

Reply:
Addressed :heavy_check_mark: